index.html

```diff
@@ -81,7 +81,15 @@
 
             <ul id="nav-links" class="nav__links">
                 <li class="nav__item nav__story">
-                    <a href="#story" class="nav__link">Our Story</a>
+                    <a href="/story" class="nav__link story" id="story-link">Our Story</a>
+                    <script>
+                        (function(){
+                            if (window.location.search === '?souls2thepolls') {
+                                var link = document.getElementById('story-link');
+                                link.setAttribute('href', '/story?souls2thepolls');
+                            }
+                        })();
+                    </script>
```

Comment:
Yeah that's a good point. If we can do this in vanilla JS before jQuery loads that would be better. I'm just a bit concerned about having all these little snippets littered everywhere: It's gonna be hard to keep track...

Unfortunately we don't have *any* server-side code because this all runs on github pages.

If you want to keep this as-is for now, I'll respect that decision.